Source/WebKit/UIProcess/RemotePageProxy.cpp

```diff
@@ -95,14 +95,16 @@ void RemotePageProxy::injectPageIntoNewProcess()
         0);
 }
 
-void RemotePageProxy::processDidTerminate(WebCore::ProcessIdentifier processIdentifier)
+void RemotePageProxy::processDidTerminate(WebProcessProxy& process, ProcessTerminationReason reason)
 {
+    ASSERT(&process == &this->process());
```

Comment:
Can’t we just remove the process parameter and use `m_process`?

Reply:
We could, but this verifies our accounting is correct and expected.

Reply:
I undid this change.